roles/pulp_common/tasks/install.yml

```diff
@@ -216,6 +216,8 @@
         group: '{{ pulp_group }}'
         recurse: true
         follow: false
+      tags:
+        - molecule-idempotence-notest
```

Comment:
Umm, this task should be idempotent.

Or we should only run it under certain conditions, or only modify certain files under it.

This may be an existing bug outside of the scope of this PR, in which case your solution is a good workaround. But if it is introduced by the changes made, we should fix it.

Reply:
I think this is being modified because:
galaxy_ng installs pulp-ansible that installs ansible,
when ansible is installed it creates $HOME/.ansible directory

Reply:
Hmm, if it is running as the pulp user account, it would run as the pulp user group. So it should have the correct group on it.

Honestly, I am not too concerned about this. If it is only ever for ansible galaxy purposes.